client/my-sites/site-logs/components/site-logs-toolbar/date-time-picker.tsx

```diff
@@ -21,20 +25,31 @@ export function DateTimePicker( { value, onChange, gmtOffset = 0, max, min, ...r
 	const moment = useLocalizedMoment();
 
 	const onDateChange = ( event: React.ChangeEvent< HTMLInputElement > ) => {
-		const newDate = moment( event.currentTarget.value, INPUT_DATE_FORMAT );
+		const newDate = moment( event.currentTarget.value, INPUT_DATE_FORMAT )
+			.utcOffset( gmtOffset * 60, true )
+			.utcOffset( value.utcOffset() );
```

Comment:
Why are we calling `utcOffset` twice here? Based on `value` definition below, shouldn't the second chaining call be `format` instead?

Reply:
I think the second `.utcOffset` is necessary to convert the moment date back to the users timezone to return to the parent component. `.format` it seems is only used for display purpose on the input fields but doesn't have an effect on the timezone.

Reply:
Below we're using `.format()` to convert the moment object into a string, because the interface for the `<input>` is string based (and it has to be in a very specific format).

The first `.utcOffset()` has two arguments, notice the `true` as the second argument. Since the `<input>` doesn't have any timezone information it's telling the moment object that it should interpret itself as being in the `gmtOffset` timezone.

The second call to `.utcOffset()` doesn't have the second argument, and it means that the moment object should change it's internal representation to `value`'s timezone.